ultradns-rest/src/main/java/denominator/ultradns/UltraDNSRestProvider.java

```diff
@@ -71,16 +86,35 @@ public String url() {
    */
   @Override
   public Map<String, Collection<String>> profileToRecordTypes() {
-    Map<String, Collection<String>>
-            profileToRecordTypes =
-            new LinkedHashMap<String, Collection<String>>();
-    profileToRecordTypes.put("geo",
-            Arrays
-                    .asList("A", "AAAA", "CNAME", "HINFO", "MX", "NAPTR", "PTR", "RP",
-                            "SRV", "TXT"));
-    profileToRecordTypes.put("roundRobin",
-            Arrays.asList("A", "AAAA", "HINFO", "MX", "NAPTR", "NS", "PTR", "RP",
-                    "SPF", "SRV", "TXT"));
+    Map<String, Collection<String>> profileToRecordTypes = new LinkedHashMap<String, Collection<String>>();
+
+    profileToRecordTypes.put("geo", Arrays.asList(
+            ResourceTypes.A.name(),
+            ResourceTypes.AAAA.name(),
+            ResourceTypes.CNAME.name(),
+            ResourceTypes.HINFO.name(),
+            ResourceTypes.MX.name(),
+            ResourceTypes.NAPTR.name(),
+            ResourceTypes.PTR.name(),
+            ResourceTypes.RP.name(),
+            ResourceTypes.SRV.name(),
+            ResourceTypes.TXT.name()
+    ));
+
+    profileToRecordTypes.put("roundRobin", Arrays.asList(
+            ResourceTypes.A.name(),
```

Comment:
` profileToRecordTypes.put(String, Collection<String>)` - is getting used only twice which is predefined. If we create a separate method to add values to a Map it won't beneficial in terms of code reuse & decrement of lines of code.
As it is has a predefined number of usage, keeping it as it is.